src/components/NewGalaxyModal.js

```diff
@@ -0,0 +1,193 @@
+            div({ style: { flex: '1', lineHeight: '1.5rem', minWidth: 0, display: 'flex' } }, [
+              span({ style: { marginRight: '0.5rem', marginTop: '0.5rem' } }, [icon('info-circle', { size: 25, color: colors.accent() })]),
+              div([
+                span({ style: styles.headerText }, ['Set up duration']),
+                div({ style: { lineHeight: '24px' } }, [
+                  div(['Creating a cloud environment for Galaxy takes ', span({ style: { fontWeight: 600 } }, ['8-10 minutes.'])]),
+                  div(['You can navigate away, and we will notify you when it\'s ready. ']),
+                ]),
+                div({ style: { ...styles.headerText, marginTop: '0.5rem' } }, ['Continuation cost']),
+                div({ style: { lineHeight: '18px' } }, [
+                  div(['Please delete the cloud environment when finished; it will']),
+                  div(['continue to ', span({ style: { fontWeight: 600 } }, ['incur charges ']), 'if it keeps running.'])
+                ])
+              ])
+            ])
+          ])
+        ])
+      ])
+    }
+
+    const renderDeleteWarning = () => {
+      return div({ style: { lineHeight: '22px' } }, [
+        div({ style: { marginTop: '0.5rem' } }, [
+          'Deleting your Cloud Environment will also ',
+          span({ style: { fontWeight: 600 } }, ['delete any files on the associated hard disk']),
+          ' (e.g. results files). Double check that your workflow results were written to ',
+          'the data tab in the workspace before clicking “Delete”'
+        ]),
+        div({ style: { marginTop: '1rem' } }, ['Deleting your Cloud Environment will stop your ',
+          'running Galaxy application and all associated costs. You can create a new Cloud Environment ',
+          'for Galaxy later, which will take 8-10 minutes.'])
+      ])
+    }
+
+    const renderDefaultCase = () => {
+      return h(Fragment, [
+        div(['Environment consists of an application and cloud compute.']),
+        div({ style: { paddingTop: '0.5rem' } }, [
+          div({ style: { ...styles.whiteBoxContainer, marginTop: '0.5rem' } }, [
+            div([
+              div({ style: styles.headerText }, ['Environment Settings']),
+              ul({ style: { paddingLeft: '1rem', lineHeight: 1.5 } }, [
+                li({ style: { marginTop: '1rem' } }, [
+                  'Galaxy version xxx'
+                ]),
+                li({ style: { marginTop: '1rem' } }, [
+                  'Cloud Compute size of ', span({ style: { fontWeight: 600 } },
+                    [`${getMachineDetails().cpu} CPUS, ${getMachineDetails().memory}  GB of memory, 50 GB disk space`])
+                  //TODO: Define the disk space using DEFAULT_DISK_SIZE from the mob_pd branch
+                ]),
+                li({ style: { marginTop: '1rem' } }, [
+                  'Running cloud compute costs ',
+                  span({ style: { fontWeight: 600 } }, '$0.19 per hr')
```

Comment:
this is the cost that was in the mocks - no calcualtion done

Reply:
I know we want things to be constant, but I don't know if we know for certain that this number is correct. However, we don't need that info in this PR. Ticket for cost info is here: https://broadworkbench.atlassian.net/browse/SATURN-1843

Reply:
I've added a todo comment and changed this to '0.00' so that it is more obvious this will need to be changed.